lighthouse-core/audits/unused-css-rules.js

```diff
@@ -33,19 +35,27 @@ class UnusedCSSRules extends Audit {
       description: 'Site does not have more than 10% unused CSS',
       helpText: 'Remove unused rules from stylesheets to reduce unnecessary ' +
           'bytes consumed by network activity. [Learn more](https://developers.google.com/speed/docs/insights/OptimizeCSSDelivery)',
-      requiredArtifacts: ['CSSUsage', 'Styles', 'URL']
+      requiredArtifacts: ['CSSUsage', 'Styles', 'URL', 'networkRecords']
     };
   }
 
   /**
    * @param {!Array.<{header: {styleSheetId: string}}>} styles The output of the Styles gatherer.
+   * @param {!Array<NetworkRecord>} networkRecords
    * @return {!Object} A map of styleSheetId to stylesheet information.
    */
-  static indexStylesheetsById(styles) {
+  static indexStylesheetsById(styles, networkRecords) {
+    const indexedNetworkRecords = networkRecords
+        .filter(record => /css/.test(record.mimeType))
```

Comment:
rather than mimeType we might want to use `record.resourceType()`: 
https://github.com/ChromeDevTools/devtools-frontend/blob/b84fc38d5618d27b6a5b50d3948cb31a38488bfa/front_end/common/ResourceType.js#L183